about.html

```diff
@@ -0,0 +1,188 @@
+                        <li><a id="joinicon" href="#"> Join</a></li>
+                        <li><a id="sponsericon" href="#"> Sponser</a></li>
+                        <li><a id="latesticon" href="#"> Latest</a></li>
+                        <li><a href="#"><button class="nav-bar-btn">Start Now!</button></a></li>
+                    </ul>
+                </div>
+        
+             
+             </nav>
+        </div>
+     </div>
+
+     <!--About Main home page section start from here -->
+    <div class="contianer-fluid about-page-fluid-container">
+
+         <!-- Mobile navbar start from here -->
+        <div class="container-fluid menu-bar-fluid">
+            <div class="row">
+                <div id="wrap" class=" wrap">
+                    <div id="bar1" class="menu"></div>
+                    <div id="bar2" class="menu"></div>
+                    <div id="bar3" class="menu"></div>
+                </div>
+            </div>
+
+            <div id="navlistdisplay2" class="row">
+                <ul class="nav-list-mobile">
+                    <li><a  href="./index.html"> Home</a></li>
+                    <li><a id="logoiconabout" href="#logo-detail-section"> Logo Detail</a></li>
+                    <li><a id="pastbatchiconabout" href="#past-Batch-section"> Past Batch Info</a></li>
+                    <li><a id="partnericonabout" href="#partner-section">Our Partners</a></li>
+                    <li><a href="./index.html">Latest</a></li>
+                </ul>
+            </div>
+        </div>
+
+      
+
+        <div class="container about-home-container">
+            <div class="row about-heading-1">
+                <h1>"Welcome, Students!"</h1>
+            </div>
+            <div class="row about-heading-2">
+                <h2>WEB DEVELOPMENT<br> COURSE BATCH 2023</h2>
+            </div>
+            <div class="row about-para-heading">
+                <p>A joyful celebration is going to start soon By the opening of classes. We have Lecturer from well reputed Universities all over the World Thereby, creating a change in education environment by giving admission to students from 180 countries.</p>
+            </div>
+            <div class="row about-heading-3">
+                <h3>Please Contact Us through Email, if you have any question about Webly Education.</h3>
+            </div>
+            <div class="row about-heading-4">
+                <h4>WeblyEducationofficial@gmail.com</h4>
+            </div>
+
+        </div>
+    </div>
+
+        <!-- Logo detail section start from here -->
+        <div id="logo-detail-section" class="contianer-fluid logo-section">
+            <div class="container">
+                <div class="row">
+                    <div class="col-12 d-flex justify-content-center logo-text">Webly-Education Logo 2023</div>
+                </div>
+                <div class="row d-flex justify-content-center">
+                    <div class="col-1 logo-text-line">line</div>
```

Comment:
- [ ] Please this part of your project is not responsive to a media screen of width `375px` so make sure you fix it according to the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

![image](https://user-images.githubusercontent.com/92303891/218156881-4ac7950f-79bf-4ecc-a074-abe06536c6f8.png)

Reply:
![image](https://user-images.githubusercontent.com/116083031/218163894-22c64ff5-2a4d-4bfa-9b7e-35bcffecf3e9.png)
Please kindly see this part is also responsive as well